doc/maintainers/team_process.md

```diff
@@ -178,8 +178,72 @@ maintainers team makes no guarantees about regularly checking them.
 now the most important thing is to direct users to a single destination.*
 
 
-## Old issue closing policy
+## Issue closing policy
 
-We have yet to settle on a precise guide for closing out old issues, but, for
-now, if you happen across something that could easily be marked as resolved,
-please do so.
+We need to strike a balance in issue handling between keeping a maintainable
+set of issues&mdash;thereby making the lives of maintainers, contributors, and
+users easier&mdash;and allowing legitimate issues to remain open.
+
+The policy for closing an issue depends entirely on the type of issue we're
+looking at.
+
+When closing an issue, please provide a brief explanation for why the issue was
+closed, such as a reference to a PR, a comment about lack of clarification,
+etc.
+
+### Discussion
```

Comment:
I have a relatively basic rule for PR vs issue: if it will take longer to open an issue than a PR, open the PR. In this case, describing the changes I'd like to make to the policy would take longer than opening up the PR with the changes. Similarly, PRs for fixing typos are easier than issues about the typos. Even minor feature requests sometimes fall into this category.

Reply:
I'm adding a section to the document with this "basic rule".